### DISCUSSION TITLE
B站小黄脸和酷安表情

### DISCUSSION THREAD

**Discussion by w0oo** [walinejs/waline#1198] [Ideas]
地址：
酷安表情：https://github.com/s-019/CEW

cdn：https://fastly.jsdelivr.net/gh/s-019/CEW/

b站小黄脸：https://github.com/s-019/W-b

cdn：https://fastly.jsdelivr.net/gh/s-019/W-b/

或者

https://w-b-iota.vercel.app（直接放上去就可以了）

哦对了，代替掉原本的jsd加速：https://fastly.jsdelivr.net/gh/自己名/你仓库/

**Comment by Mister-Hope** [comment]
你可以pr到walinejs/emoji上，欢迎

**Reply by w0oo** [reply]
不会搞😴

**Comment by Zengzzhao** [comment]
> 地址： 酷安表情：https://github.com/s-019/CEW
> 
> cdn：https://fastly.jsdelivr.net/gh/s-019/CEW/
> 
> b站小黄脸：https://github.com/s-019/W-b
> 
> cdn：https://fastly.jsdelivr.net/gh/s-019/W-b/
> 
> 或者
> 
> [https://w-b-iota.vercel.app（直接放上去就可以了）](https://w-b-iota.vercel.app%EF%BC%88%E7%9B%B4%E6%8E%A5%E6%94%BE%E4%B8%8A%E5%8E%BB%E5%B0%B1%E5%8F%AF%E4%BB%A5%E4%BA%86%EF%BC%89)
> 
> 哦对了，代替掉原本的jsd加速：[https://fastly.jsdelivr.net/gh/自己名/你仓库/](https://fastly.jsdelivr.net/gh/%E8%87%AA%E5%B7%B1%E5%90%8D/%E4%BD%A0%E4%BB%93%E5%BA%93/)
![PixPin_2025-01-15_13-04-30](https://github.com/user-attachments/assets/ba4a008f-46f6-420c-9a87-44c2df232de2)
![PixPin_2025-01-15_13-04-46](https://github.com/user-attachments/assets/134ffc1d-6cba-4970-b7dd-d75e8bc3f053)
为啥我引用后还是没有，这个配置有问题吗

---
> Address: Coolan Expression: https://github.com/s-019/CEW
>
> cdn: https://fastly.jsdelivr.net/gh/s-019/CEW/
>
> Little yellow face at station b: https://github.com/s-019/W-b
>
> cdn: https://fastly.jsdelivr.net/gh/s-019/W-b/
>
> or
>
> [https://w-b-iota.vercel.app (Just put it directly)](https://w-b-iota.vercel.app%EF%BC%88%E7%9B% B4%E6%8E%A5%E6%94%BE%E4%B8%8A%E5%8E%BB%E5%B0%B1%E5%8F%AF%E4%BB%A5%E4%BA%86% EF%BC%89)
>
> Oh, by the way, replace the original jsd acceleration: [https://fastly.jsdelivr.net/gh/ownname/yourwarehouse/](https://fastly.jsdelivr.net/gh/%E8%87% AA%E5%B7%B1%E5%90%8D/%E4%BD%A0%E4%BB%93%E5%BA%93/)
![PixPin_2025-01-15_13-04-30](https://github.com/user-attachments/assets/ba4a008f-46f6-420c-9a87-44c2df232de2)
![PixPin_2025-01-15_13-04-46](https://github.com/user-attachments/assets/134ffc1d-6cba-4970-b7dd-d75e8bc3f053)
Why is it still missing after I quoted it? Is there something wrong with this configuration?
